### PR TITLE
Update monochrome tutorial to latest API

### PR DESCRIPTION
**Pull request by Maldus512**

### CODE REVIEW

File: _posts/2019-05-06-oled.md [hunk 1]
```diff
@@ -159,8 +171,12 @@ And then the last extra piece you need is to select the MONO theme after initial
 ```c
     oledInit();
     
+    static lv_disp_buf_t disp_buf;    /*Descriptor of a display buffer */
+    static lv_disp_drv_t disp_drv;    /*Descriptor of a display driver*/
+    static lv_color_t gbuf[BUFSIZE];  /*Memory area used as display buffer */
+    
     lv_init();
-    lv_disp_drv_t disp_drv;                         /*Descriptor of a display driver*/
+    lv_disp_buf_init(&disp_buf, gbuf, NULL, BUFSIZE); /* BUFSIZE can be smaller than the actual display */
```
Comment (embeddedt) [inline _posts/2019-05-06-oled.md]: `BUFSIZE` is a bit dangerous. People are likely to not read part of this tutorial and just copy the code, and when they get a message that `BUFSIZE` is not declared, they'll look it up and find that POSIX has a `BUFSIZ` variable.

Possibly it should be named `DISP_BUF_SIZE` instead?

**Review by embeddedt — APPROVED** [review]
One minor concern

File: _posts/2019-05-06-oled.md (hunk 1 above)
Comment (Maldus512) [reply _posts/2019-05-06-oled.md]: Agreed, I renamed it and added an extra explanation for good measure.

**Comment by kisvegabor** [comment]
Looks good to me! Is it ready to merge?

**Comment by Maldus512** [comment]
> Looks good to me! Is it ready to merge?

As far as I'm concerned it's done, you can proceed with the merge

**Comment by kisvegabor** [comment]
Merged, thank you very much!

**Pull request merged**